magellan-flow-sample/src/main/java/com/wealthfront/magellan/sample/MainActivity.kt

```diff
@@ -0,0 +1,21 @@
+package com.wealthfront.magellan.sample
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.wealthfront.magellan.lifecycle.setContentScreen
+
+class MainActivity : AppCompatActivity() {
+
+  private val expeditionProvider = ExpeditionProvider
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentScreen(expeditionProvider.expedition)
+  }
+
+  override fun onBackPressed() {
```

Comment:
Aww we can't do this automatically, can we?

Reply:
unfortunely, no :(